model/src/main/java/org/mskcc/cmo/metadb/model/MetadbRequest.java

```diff
@@ -365,8 +365,8 @@ public void updateRequestMetadataByMetadata(RequestMetadata requestMetadata)
         Map<String, Object> metadataMap =
                 mapper.readValue(requestMetadata.getRequestMetadataJson(), Map.class);
 
-        this.igoRequestId = String.valueOf(metadataMap.get("requestId"));
```

Comment:
now I understand what you were referring to before - so here is where we would be processing something that's being published from LimsRest which would still have the former property names. If you want to generalize this then we can add helper methods to resolve the values for requestId and gene panel/recipe

You can refer to the helper methods added to SampleDataFactory. 

But basically the helper method should:

1. check if the map contains `requestId` and use the value if that property is present
2. if (1) fails then check for `igoRequestId` and use that value instead